Project-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,52 @@
 import React from 'react';
 import './Display.css';
+import ShapeSquare from './ShapeSquare';
+import RectangleShape from './RectangleShape';
+import RectoSquareShape from './RectoSquareShape';
+import NumberButton from "../ButtonComponents/NumberButton"
+
+
+const CalculatorDisplay =()=>{
+
+    const data = {
```

Comment:
You chose an interesting way of approaching this problem. Your naming conventions are unconventional, but I can clearly see how you worked out the project. It appears you went at it from a style first perspective rather than function. There is nothing wrong with this, but it might make creating functionality challenging or the code harder to interpret. Comments can easily solve this, but I would also suggest trying to think about solving the functionality problem first or at the same time you consider styling.